static/styles/settings.css

```diff
@@ -1581,13 +1581,14 @@ body:not(.night-mode) #settings_page .custom_user_field .datepicker {
     margin-top: 10px;
 }
 
-#do_deactivate_self_button .loader {
+.confirm_dialog_yes_button .loader {
     display: none;
     vertical-align: top;
     position: relative;
     height: 30px;
     margin-top: -10px;
     top: 5px;
+    width: 30px;
```

Comment:
I have to add this to avoid an unwanted transition, where button just widens for a moment then resizes to its original size.

Reply:
That looks right.